portfolio/pom.xml

```diff
@@ -33,11 +33,6 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
-    <dependency>
-      <groupId>com.google.appengine</groupId>
```

Comment:
Is this still needed for other APIs like login or datastore?

Reply:
I accidentally added that dependency twice so I was removing one of them.